config/app.default.php

```diff
@@ -9,7 +9,7 @@
      * Development Mode:
      * true: Errors and warnings shown.
      */
-    'debug' => true,
+    'debug' => env('DEBUG', true),
```

Comment:
BTW, I'm worrying that the debug bit will work as expected. The env variables come as strings.

```php
<?php
$debug = 'false';
if ($debug) {
    echo "No---\n";
}
```